apps/src/lib/util/StatsigReporter.js

```diff
@@ -53,11 +56,11 @@ class StatsigReporter {
   }
 
   // Utilizes Statsig's function for updating a user once we've recognized a sign in
-  async setUserProperties(userId, userType) {
+  async setUserProperties(userId, userType, enabledExperiments) {
```

Comment:
additionally, allow experiment list to be passed through from current user redux